extension/addreactionparticipants.js

```diff
@@ -0,0 +1,60 @@
+const addReactionParticipants = currentUser => {
+	$('.comment-reactions').each((index, reactionsContainer) => {
```

Comment:
Hmm, I can't remember if the socket refreshes the comment's reactions or all reactions. I can take a look.